### PR TITLE
Dingo_kinova urdf files

### PR DESCRIPTION
**Pull request by TomasMerva**

### CODE REVIEW

**Comment by maxspahn** [comment]
Hi @TomasMerva,

Thanks for the addition of the dingo kinova robot.

I'll check it today to proceed to accepting the PR.

Best,
@maxspahn

File: robotmodels/dingo_kinova/urdf/dingo_kinova.urdf [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
+    </inertial>
+  </link>
+  <link name="base_link_y">
+    <inertial>
+      <mass value="1.0"/>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
+    </inertial>
+  </link>
```
Comment (maxspahn) [inline robotmodels/dingo_kinova/urdf/dingo_kinova.urdf:37]: I think it is clearer if these links are at the beginning of the file to reflect the order of the serial chain.

File: robotmodels/dingo_kinova/urdf/dingo_kinova.urdf [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+  <joint name="world_link" type="fixed">
+    <origin rpy="0 0 0" xyz="0 0 0"/>
+    <parent link="world"/>
+    <child link="base_link"/>
+  </joint>
+  <joint name="arm_mount_joint" type="fixed">
+    <origin rpy="0 0 0" xyz="0 0 0"/>
+    <parent link="front_b_mount"/>
+    <child link="arm_base_link"/>
+  </joint>
```
Comment (maxspahn) [inline robotmodels/dingo_kinova/urdf/dingo_kinova.urdf]: For readability, I would prefer to have the world joint at the beginning and the arm mount joint at the appropriate location before the details on the arm.

**Review by maxspahn — CHANGES_REQUESTED** [review]
hi @TomasMerva 

Thanks for the PR. I made some comments concerning the order of element in the urdf, but it looks good to me. Coulde you also add the xml model? Otherwise, ill do so.

Best,
@maxspahn

File: robotmodels/dingo_kinova/urdf/dingo_kinova.urdf [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+  <!-- ==================== -->
+  <!-- Dingo links          -->
+  <!-- ==================== -->
+  <!-- Chassis link -->
+  <link name="chassis_link">
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="../meshes/base/omni_chassis.dae"/>
```
Comment (TomasMerva) [inline robotmodels/dingo_kinova/urdf/dingo_kinova.urdf:74]: `chassis_link` uses .dae file, which is not the supported format in mujoco or yourdfpy. Should I leave it as it is or should I just use `omni_chassis_collision.stl`?

File: robotmodels/dingo_kinova/xml/dingo_kinova.xml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    <velocity joint="omni_joint_x" ctrlrange="-2.0 2.0" kv='100' ctrllimited="true" />
+    <velocity joint="omni_joint_y" ctrlrange="-2.0 2.0" kv='100' ctrllimited="true" />
+    <velocity joint="omni_joint_theta" ctrlrange="-2.0 2.0" kv='90' ctrllimited="true" />
+    <velocity joint="arm_joint_1" ctrlrange="-2.0 2.0" kv='100' ctrllimited="true" />
+    <velocity joint="arm_joint_2" ctrlrange="-2.0 2.0" kv='100' ctrllimited="true" />
+    <velocity joint="arm_joint_3" ctrlrange="-2.0 2.0" kv='90' ctrllimited="true" />
+    <velocity joint="arm_joint_4" ctrlrange="-2.0 2.0" kv='80' ctrllimited="true" />
+    <velocity joint="arm_joint_5" ctrlrange="-2.0 2.0" kv='70' ctrllimited="true" />
+    <velocity joint="arm_joint_6" ctrlrange="-2.0 2.0" kv='50' ctrllimited="true" />
+  </actuator>
```
Comment (TomasMerva) [inline robotmodels/dingo_kinova/xml/dingo_kinova.xml:88]: I autogenerated the xml files based on urdf files and copied these random controllers from the `kinova.xml` file. Is it alright or should I remove these controllers?

**Comment by TomasMerva** [comment]
Hi @maxspahn 

I have changed the order and added the XML files, however, I have two questions.

File: robotmodels/dingo_kinova/urdf/dingo_kinova.urdf (hunk 3 above)
Comment (maxspahn) [reply robotmodels/dingo_kinova/urdf/dingo_kinova.urdf:74]: Please use the `stl`-files instead.

File: robotmodels/dingo_kinova/xml/dingo_kinova.xml (hunk 4 above)
Comment (maxspahn) [reply robotmodels/dingo_kinova/xml/dingo_kinova.xml:88]: These controllers are fine. Tuning is up to the user ;)

**Comment by maxspahn** [comment]
@TomasMerva  I have verified the xml files in mujoco, and everything is fine. So Ill merge it now.

**Review by maxspahn — APPROVED** [review]

**Pull request merged**